src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurDetailLineGroupForCollector.java

```diff
@@ -320,7 +320,7 @@ protected void addOriginEntriesForCorpCardPersonalExpenseDebitLinesIfNecessary(C
         ConcurStandardAccountingExtractDetailLine firstDebitLine = debitLines.get(0);
         if (cashOffsetAdjustment.isNonZero()) {
             addOriginEntryForCorpCardPersonalExpense(
-                    entryConsumer, firstDebitLine, collectorHelper.getCashOffsetObjectCode(), cashOffsetAdjustment);
+                    entryConsumer, firstDebitLine, collectorHelper.getPaymentOffsetObjectCode(), cashOffsetAdjustment);

```

Comment:
OK.  I will rename the items in this class that have the term "cashOffset" in their names to the term "paymentOffset" taking into account proper camel casing where that term is used.